### PR TITLE
Node.js bindings: Add bindings for gpio_open_by_label and gpio_open_raw

### PR DESCRIPTION
**Pull request by nagineni**
This adds missing bindings for gpio_open_by_label and gpio_open_raw.

Signed-off-by: Sudarsana Nagineni <sudarsana.nagineni@intel.com>

### CODE REVIEW

**Comment by nagineni** [comment]
@edersondisouza @cabelitos @gabrielschulhof Could you please have a look?

File: bindings/nodejs/src/functions/gpio.cc [hunk 1]
```diff
@@ -53,42 +91,26 @@ NAN_METHOD(bind_sol_gpio_open) {
...
-        return;
-    } else {
-        if (callback) {
-            delete callback;
-            hijack_unref();
-        }
-        delete gpio_data;
-    }
+    uint32_t pin = info[0]->Uint32Value();
+    OPEN_GPIO(sol_gpio_open_raw, pin, info[1]->ToObject());
```
Comment (gabrielschulhof) [inline bindings/nodejs/src/functions/gpio.cc]: Why don't you move ```->ToObject()``` into the macro definition?

File: bindings/nodejs/src/functions/gpio.cc [hunk 2]
```diff
@@ -35,6 +35,44 @@ class SolGpio : public JSHandle<SolGpio> {
...
 
+#define OPEN_GPIO(functionname, pin, jsGPIOConfig) \
+    do { \
+        sol_gpio_config config; \
+        sol_gpio *gpio = NULL; \
+\
+        sol_gpio_data *gpio_data = new sol_gpio_data; \
+        gpio_data->callback = NULL; \
+\
+        if (!c_sol_gpio_config(jsGPIOConfig, gpio_data, &config)) { \
```
Comment (gabrielschulhof) [inline bindings/nodejs/src/functions/gpio.cc]: Here, you would do ```jsGPIOConfig->ToObject()``` and you wouldn't have to do it three different times in the code.

**Comment by gabrielschulhof** [comment]
No big deal, just a minor suggestion.

File: bindings/nodejs/src/functions/gpio.cc (hunk 2 above)
Comment (nagineni) [inline bindings/nodejs/src/functions/gpio.cc]: I will fix it.

**Pull request closed without merging**